hack/lib/strimzi.bash

```diff
@@ -48,9 +48,7 @@ function install_strimzi_cluster {
           offsets.topic.replication.factor: 3
           transaction.state.log.replication.factor: 3
           transaction.state.log.min.isr: 2
-          inter.broker.protocol.version: "3.0"
-          log.message.format.version: "3.0"
-          auto.create.topics.enable: "false"
```

Comment:
is this setting gone? it's nice to have that one to make sure we're actually creating topics.

Reply:
>           auto.create.topics.enable: "false"

jikes! I removed that by accident.

Added it back